third_party/config/monitoring/logging/elasticsearch/elasticsearch.yaml

```diff
@@ -94,6 +94,7 @@ spec:
         name: elasticsearch-logging
         resources:
           limits:
+            memory: 1000Mi
```

Comment:
I am worried that 1gig might be too little for Elastic Search. In these memory constraint environments, it might be better to not install Elastic Search at all and undo this change.

Reply:
I'm open to changing it - what value would you be more comfortable with?

Reply:
I don't have a good answer to that, but the recommendation from ES is to have as much as you can afford - https://qbox.io/support/article/choosing-a-size-for-nodes